LPS-120432 Remove unnecessary useMemo occurrences and improve the metrics pages performance

albinojunior opened the pull request.

Commit:
LPS-120432 Remove unnecessary useMemo occurrences and improve the metrics pages performance

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

albinojunior commented:
ci:test:workflow

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest(master)/3522/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-10.liferay.com//userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3522/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:x: ci:test:workflow - 90 out of 99 jobs passed in 2 hours 13 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/3428a35a1038bb9b5cb36391be722f78707b7f50">3428a35a1038bb9b5cb36391be722f78707b7f50</a></p><h4>Copied in Private Modules Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal-ee/tree/master-private">master-private</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal-ee/commit/df1236053a2097d63e354600635a9bf8b08e5a99">df1236053a2097d63e354600635a9bf8b08e5a99</a></p><details><summary><strong>ci:test:workflow - 90 out of 99 jobs PASSED</strong></summary><div><h4>9 Failed Jobs:</h4><ul><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest(master)/3522/"><strike><strong>test-portal-acceptance-pullrequest(master)</strong></strike></a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/460906/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</strong></strike></a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/299051/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/12</strong></strike></a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/191327/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/37</strong></strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/436199/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/48</strong></strike></a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400510/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/49</strong></strike></a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/401370/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/56</strong></strike></a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/299054/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/80</strong></strike></a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/431424/"><strike><strong>test-portal-acceptance-pullrequest-batch(master)/modules-integration-mysql57-jdk8</strong></strike></a></li></ul></div><details><summary><strong>90 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/428317/">test-portal-acceptance-pullrequest-batch(master)/central-requirements-jdk8</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/401367/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352788/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/10</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/392551/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/11</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/431420/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/13</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/439806/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/14</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/436969/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/15</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/322188/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/16</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/428398/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/17</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291985/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/18</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/191326/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/19</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/450898/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/428397/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/20</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291984/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/21</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/401366/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/22</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/431419/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/23</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352487/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/24</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291983/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/25</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/431418/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/26</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/401365/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/27</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/450897/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/28</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/439504/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/29</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352488/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/428399/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/30</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352789/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/31</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/392552/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/32</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/410835/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/33</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/299052/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/34</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429995/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/35</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/460907/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/36</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/401368/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/38</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291986/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/39</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/428316/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/436970/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/40</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/439506/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/41</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/431421/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/42</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/439807/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/43</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/450899/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/44</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352489/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/45</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/322189/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/46</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/304041/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/47</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/439505/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/431423/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/50</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/439809/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/51</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352491/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/52</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291988/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/53</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/191329/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/54</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/460909/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/55</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/436972/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/57</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/322191/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/58</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/439508/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/59</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/304040/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/191328/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/60</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/431422/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/61</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352490/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/62</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/436971/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/63</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/428400/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/64</a></li><li><a href="https://test-1-23.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/291987/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/65</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429996/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/66</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/460908/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/67</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/439507/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/68</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/322190/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/69</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352787/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/439810/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/70</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/439509/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/71</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/304042/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/72</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352791/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/73</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/436201/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/74</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352492/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/75</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/436973/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/76</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/428320/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/77</a></li><li><a href="https://test-1-12.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/322192/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/78</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400512/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/79</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400509/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/8</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/299050/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/9</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/392553/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/0</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/410836/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/1</a></li><li><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/299053/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/2</a></li><li><a href="https://test-1-15.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/304039/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/3</a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/436200/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/4</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400511/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/5</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/352790/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/6</a></li><li><a href="https://test-1-13.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/401369/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/7</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/439808/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/428318/">test-portal-acceptance-pullrequest-batch(master)/functional-upgrade-tomcat90-mysql57-jdk8/9</a></li><li><a href="https://test-1-1.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/392554/">test-portal-acceptance-pullrequest-batch(master)/js-unit-jdk8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/429997/">test-portal-acceptance-pullrequest-batch(master)/modules-semantic-versioning-jdk8</a></li><li><a href="https://test-1-7.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/410837/">test-portal-acceptance-pullrequest-batch(master)/modules-unit-jdk8</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/428401/">test-portal-acceptance-pullrequest-batch(master)/semantic-versioning-jdk8</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/428319/">test-portal-acceptance-pullrequest-batch(master)/unit-jdk8</a></li></ul></details></details><h5>For more details click <a href="https://test-1-10.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3522/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><h5><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/460906/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/0</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/460906//consoleText">AXIS_VARIABLE=0,label_exp=!master #460906</a><ol><li><div><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/460906//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_Kaleodesigner_ViewSourceXMLNotErasedWhenInvalid_">LocalFile.Kaleodesigner#ViewSourceXMLNotErasedWhenInvalid</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.Kaleodesigner_ViewSourceXMLNotErasedWhenInvalid/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/0/0/LocalFile.Kaleodesigner_ViewSourceXMLNotErasedWhenInvalid/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/0/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/299051/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/12</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/299051//consoleText">AXIS_VARIABLE=0,label_exp=!master #299051</a><ol><li><div><a href="https://test-1-22.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/299051//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_WorkflowMetrics_FilterAllItemsListByAssigneeName_">LocalFile.WorkflowMetrics#FilterAllItemsListByAssigneeName</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/12/0/LocalFile.WorkflowMetrics_FilterAllItemsListByAssigneeName/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/12/0/LocalFile.WorkflowMetrics_FilterAllItemsListByAssigneeName/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/12/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/191327/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/37</a></h5><div><h6>Job Results:</h6><p>1 Test Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/191327//consoleText">AXIS_VARIABLE=0,label_exp=!master #191327</a><ol><li><div><a href="https://test-1-18.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/191327//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_WorkflowMessageBoards_RejectMessageBoardsThreadPG_">LocalFile.WorkflowMessageBoards#RejectMessageBoardsThreadPG</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/37/0/LocalFile.WorkflowMessageBoards_RejectMessageBoardsThreadPG/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/37/0/LocalFile.WorkflowMessageBoards_RejectMessageBoardsThreadPG/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/37/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/436199/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/48</a></h5><div><h6>Job Results:</h6><p>15 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/436199//consoleText">AXIS_VARIABLE=1,label_exp=!master #436199</a><ol><li><div><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=1,label_exp=!master/436199//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_WorkflowMessageBoards_ApproveMessageBoardsThreadPG_">LocalFile.WorkflowMessageBoards#ApproveMessageBoardsThreadPG</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/48/1/LocalFile.WorkflowMessageBoards_ApproveMessageBoardsThreadPG/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/48/1/LocalFile.WorkflowMessageBoards_ApproveMessageBoardsThreadPG/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/48/1/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li><div><h5><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/400510/">test-portal-acceptance-pullrequest-batch(master)/functional-tomcat90-mysql57-jdk8/49</a></h5><div><h6>Job Results:</h6><p>14 Tests Passed.<br/>1 Test Failed.</p></div><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/400510//consoleText">AXIS_VARIABLE=0,label_exp=!master #400510</a><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-batch(master)/AXIS_VARIABLE=0,label_exp=!master/400510//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_WorkflowUsecase_XSSScriptWebContent_">LocalFile.WorkflowUsecase#XSSScriptWebContent</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/49/0/LocalFile.WorkflowUsecase_XSSScriptWebContent/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/49/0/LocalFile.WorkflowUsecase_XSSScriptWebContent/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/test-1-10/1599597838583/test-portal-acceptance-pullrequest(master)/3522/functional-tomcat90-mysql57-jdk8/49/0/jenkins-console.txt.gz">Console Output</a></div></li></ol></div></li></ol></div></li><li>...</li></ol></details></html>

liferay-continuous-integration commented:
<strong>Jenkins Build:</strong><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest(master)/3522/">test-portal-acceptance-pullrequest(master)#3522</a><br /><strong>Jenkins Report:</strong><a href="https://test-1-10.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/3522/jenkins-report.html">jenkins-report.html</a><br /><strong>Jenkins Suite:</strong> workflow<br /><strong>Pull Request:</strong><a href="https://github.com/albinojunior/liferay-portal/pull/182">albinojunior#182</a><br /><strong>Spira Release:</strong><a href="https://liferay.spiraservice.net/16/Release/1580.aspx">/Liferay DXP 7.3/Pull Request/ci:test:workflow</a><br /><strong>Spira Release Build:</strong><a href="https://liferay.spiraservice.net/16/Build/38085.aspx">master - albinojunior > albinojunior - PR#182 - 2020-09-08[13:43:58]</a><br /><strong>Spira Jenkins Build:</strong><a href="https://test-1-20.liferay.com/job/publish-spira-report/3933/">publish-spira-report#3933</a><br />

kyle-miho committed:
LRQA-63923 Adjust expect output file

Commit:
LPS-121919 - Migrate management toolbar v2 usages in style-book

Commit:
LPS-128186 Do not export jackson from core, required-dependencies had already deployed the jar as module. This may cause a "uses" constraint violation which is discovered when upgrade Log4J.

tinatian committed:
LPS-125549 Extract logic into Log4jConfigUtil#shutdownLog4J

tinatian committed:
LPS-125549 Use string to avoid importing log4j package

tinatian committed:
LPS-125549 SF, update JavaXMLSecurityCheck suppressions.

hhuijser committed:
LPS-105380 Use c:choose tag instead of if/else statement

istvansajtos committed:
LPS-124886 portal-search-web: We consider excludedEntryClassNames set under Engine Helper configuration when determining searchable assets by reusing SearchableEntryClassNamesProvider

istvansajtos committed:
LPS-124886 portal-search: Consider excludedEntryClassNames set under Engine Helper configuration when determining searchable assets

tinatian committed:
LPS-125549 SF, simpify

Commit:
COMMERCE-4013 fix accept language

ealonso committed:
LPS-128286 Removes ddm-journal-article from ddm since it was used only for web content and now in web content we are using the new data engine

Commit:
COMMERCE-4787 Remove quarantine

Commit:
COMMERCE-5699 Remove quarantine

brandizzi committed:
LPS-124886 portal-search-web: Test for AssetEntriesSearchFacet and TypeFacetPortletPreferencesImplTest

https://issues.liferay.com/browse/LPS-124886

brandizzi committed:
LPS-124886 portal-search: Test for SearchableAssetClassNamesProvider.

https://issues.liferay.com/browse/LPS-124886

tinatian committed:
LPS-125549 Add unit test

Commit:
LPS-121871 Migrate v2 usages in document-library/document-library-web

The <clay:management-toolbar-v2 /> tag has been replaced with the
<clay:management-toolbar /> tag (which uses clay v3).

**Test plan**:
- Navigate to **Content & Data** > **Documents and Media**
- Uploading new files should work as expected
- Editing tags should work as expected
- Deleting items should work as expected
- Moving items to a new folder should work as expected
- Creating Document Types should work as expected
- Creating Metadata Sets should work as expected

Commit:
COMMERCE-5731 dispatch-service - avoid NPE, check if DispatchTaskExecutor is found

Commit:
COMMERCE-5731 dispatch-service - set end date too

ealonso committed:
LPS-128160 Concat parent structure id to field set name to make it unique since we can have a parent structure of a parent structure ...

ealonso committed:
LPS-128160 Update tests results

Commit:
COMMERCE-5731 dispatch-test - add test when missing dispatchTaskExecutor

achaparro committed:
LPS-128098 This is not longer needed. We need to define a more generic approach which covers all cases

IstvanD committed:
LPS-127860 URL in mail verification has 'nullnull' after creating a new virtual instance

izaera committed:
LPS-127990 Scope all occurrences of frontend-js-react-web npm package

izaera committed:
LPS-127990 Support namespaced npm packages in webpack federation

Note this needs https://github.com/liferay/liferay-frontend-projects/pull/408 to be merged to work.

achaparro committed:
LPS-128098 Allow generic methods in service builder

Paulovcaselli committed:
LPS-126340 Add CheckLanguageSelector test

Paulovcaselli committed:
LPS-126340 SF

Commit:
LPS-121892 - Migrate management toolbar v2 usages in my-subscriptions/my-subscriptions-web

achaparro committed:
LPS-128098 New entity to test the build of generic methods

Commit:
COMMERCE-5739 use proper delivery next iteration date

achaparro committed:
LPS-128098 build service

Commit:
COMMERCE-5699 Keep quarantine for tests that have dependencies with Accounts or Minium

achaparro committed:
LPS-128098 Tests: add generic methods

izaera committed:
LPS-127990 Use npm-scripts 37.1.1

achaparro committed:
LPS-128098 SF

achaparro committed:
LPS-128098 Build service

Commit:
LPS-127933 Fix cursor blinking in step card and EditAppSideBar

Commit:
COMMERCE-5739 fix commerce delivery subscription test

shuyangzhou committed:
LPS-128098 Generics

shuyangzhou committed:
LPS-128098 No reason to use 2 SB.

shuyangzhou committed:
LPS-128098 Stick to use SB.

pietrobocale committed:
COMMERCE-5748 Remove from quarantine CPCommerceChannels

pietrobocale committed:
COMMERCE-5748 Remove from quarantine CPCommerceChannels#CreateNewChannel

pietrobocale committed:
COMMERCE-5748 Remove from quarantine CPCommerceChannels#CreateNewNotificationTemplate

pietrobocale committed:
COMMERCE-5750 Remove from quarantine CPCommerceCatalogs

pietrobocale committed:
COMMERCE-5750 Remove from quarantine CPCommerceCatalogs#CreateNewCatalog

pietrobocale committed:
COMMERCE-5753 Remove from quarantine CommerceSmoke

pietrobocale committed:
COMMERCE-5753 Remove from quarantine CommerceSmoke#CreateNewSpecificationOptions

pietrobocale committed:
COMMERCE-5753 Remove from quarantine CommerceSmoke#PublishSimpleProduct

slnn committed:
LPS-128188 Add back the ant property to allow user to pass the property file for SampleSQLBuilder; this will be replaced later when we finish the full refactoring.

slnn committed:
LPS-128188 Segments: move the loop to the ftl so user can avoid generating these segment data when related property did not set

slnn committed:
LPS-128188 Segments: remove the csv generation for Segment, current benchmark segment test case did not need this because it reuses login

izaera committed:
LPS-127990 SF

Commit:
LPS-121862 Migrate v2 usages in asset/asset-tags-admin-web

The <clay:management-toolbar-v2 /> tag has been replaced with the
<clay:management-toolbar /> tag (which uses clay v3).

Creating, merging and deleting tags should work as expected.

**Test plan**:
- Navigate to **Categorization**>**Tags**
- Assert that you can create a tag
- Assert that you can delete the tag you created previously
- Assert that you can merge two or more tags correctly.

ambrinchaudhary committed:
LPS-121893 Migrate v2 management toolbar usages in notifications-web

victorg1991 committed:
LPS-128044 isCopyLayout thead local values should be false by default, it should only be true when setting it explicitly

ambrinchaudhary committed:
LPS-121879 Migrate v2 management toolbar usages in item-selector-taglib

Commit:
COMMERCE-5731 dispatch - rename methods to indicate they can return null

Commit:
COMMERCE-5731 dispatch-api - baseline

izaera committed:
LPS-127990 Upgrade npm-scripts to fix CI tests

Commit:
LPS-124594 As a Site Template Administrator, I want to have a one-click control for switching the propagation off and on

Commit:
COMMERCE-5699 Quarantine test fixed in different pull request

izaera committed:
LPS-127990 Change two more missing refs to @liferay/frontend-js-react-web

izaera committed:
LPS-123585 Remove bundler 3 from frontend-js-recharts

izaera committed:
LPS-127990 SF

samanzanedo committed:
LPS-128211 Checking if AssetRendered object is null to avoid a NPE

Commit:
LPS-121912 Migrate v2 usages in site/site-memberships-web

The <clay:management-toolbar-v2 /> tag has been replaced with the
<clay:management-toolbar /> tag (which uses clay v3).

Managing site memberships should work as expected.

**Test plan**:
- Navigate to **People**>**Memberships**
- Assert that you can add and delete a user
- Assert that you can asign one or several roles to a user
- Assert that you can add and delete and organization
- Assert that you can asign one or several roles to an organization
- Assert that you can create and delete a user group
- Assert that you can asign one or several roles to a user group

cgoncas committed:
LPS-124170 Rename, because this test just add an experience, no an experiment.

cgoncas committed:
LPS-124170 Test LanguageIds param is correctly added to the segment experience.

Commit:
LPS-124594 gw buildLang

ealonso committed:
LPS-128214 Set parentStructureId to 0, since now we have field sets instead of parent structures

cgoncas committed:
LPS-124170 Test corner cases too.

cgoncas committed:
LPS-124170 If the saved locales are empty, we should return all the available languages of the site.

cgoncas committed:
LPS-124170 Fix tests

cgoncas committed:
LPS-124170 Add UpdateSegmentsExperienceMVCActionCommandTest.

ruben-pulido committed:
LPS-124271 Remove adaptive media feature flag

michaelhashimoto committed:
LRCI-2000 Get build properties should use getProperty() in order to use references

michaelhashimoto committed:
LRCI-2000 Add classes for S3 Buckets & Objects

michaelhashimoto committed:
LRCI-2000 Add reference com.amazonaws:aws-java-sdk-s3:1.11.220 to jenkins-results-parser

joshuacords committed:
LPS-124886 portal-search-web: update jsp method call

https://issues.liferay.com/browse/LPS-124886

zhangyan0701 committed:
LRQA-64540 Add all database types for upgrade tests from 7310

boton committed:
LPS-128049 Add flexible height to Codemirror in Web Content

pei-jung committed:
LPS-127860 Use company.getPortalURL

Commit:
LRQA-65259 Update the locator for Publications#LIST_PUBLISHED_STATUS

Sharryxue committed:
LRQA-65040 Update the locator

Commit:
LRQA-65260 Update the locator for Publications#LIST_REVERT_BUTTON

Commit:
LPS-121912 Remove unused DefaultEventHandlers

Commit:
LRQA-64878 Remove deprecated locators

Commit:
LRQA-65261 Update the locator for Publications#LIST_REVERT_DISABLED

Commit:
LRQA-64878 Replace locators

Commit:
LRQA-64878 Rerun case on upstream and acceptance

CarlosBrichete committed:
LPS-128200 Added AuditGraphClickOnBar test and paths and macros needed

Commit:
COMMERCE-5695 Dataset Display update events now match

Commit:
COMMERCE-4013 static env utilities updated to make tests working

Commit:
LPS-125404 Hide service list as default

adolfopa committed:
LPS-127986 This will never be null

adolfopa committed:
LPS-127986 Path data definition before posting

What we really need here is a new API method to copy a data definition.
Ideally, that should use the existing copy method in the DDM service.

Jeremy-Ch committed:
LRQA-64535 Split the test to distinguish local staging and remote staging

marcoscv-work committed:
LPS-125542 - Update breaking changes with the removal of the compatibility layer

CarlosBrichete committed:
LPS-128200 Modified teardown and clear button path

boton committed:
LPS-128049 Add wrapper and add class hide when is needed but we always need the wrapper here

boton committed:
LPS-128049 Add flexible height to Codemirror in data engine fields

boton committed:
LPS-128049 Auto SF

Commit:
LPS-128254 Split the CI Acceptance Test Tasks from the Headless team

ambrinchaudhary committed:
LPS-121880 Migrate v2 usages in item-selector/item-selector-web

CarlosBrichete committed:
LPS-128200 Added AuditGraphClickOnBarDeleteOneFilter test and needed macro and path

CarlosBrichete committed:
LPS-128200 SF

Commit:
LPS-125404 Add test to the whiteboard integration

Commit:
LPS-125404 Rename for consistency

Commit:
COMMERCE-5731 dispatch-service - Rename and SF

adolfopa committed:
LPS-128019 Localizable should be positioned after Show Label

Commit:
COMMERCE-5731 dispatch-service - improve - provide error message for this case too - match to pattern in _execute to achieve consistent user reporting for all cases

brianchandotcom committed:
LPS-128098 SF

marcosapmf committed:
LPS-128257 make sure sidebar is opened after changing focused field

victorg1991 committed:
LPS-128211 Simplify

markocikos committed:
LPS-127020 Add attribute to set auto-focus on search input

markocikos committed:
LPS-127020 Add sample use of auto-focus on search input

4lejandrito committed:
LPS-128235 Only showing the error when the configuration is enabled

4lejandrito committed:
LPS-128235 As used

hhuijser committed:
LPS-105380 Partial revert

Commit:
LPS-128254 Refactor the CI Acceptance Test Tasks from the Headless team

brianchandotcom committed:
LPS-125549 SF

Commit:
LPS-127777 Consider Integer as possible type for identifiers

dgarciasarai committed:
LPS-127642 Merge actual experience with update experience, not only the priority

We were losing information on experience update like if the experience has an experiment or not and the page editor sidebar was rendering wrong

dgarciasarai committed:
LPS-127642 Fix PropTypes

dgarciasarai committed:
LPS-127642 Sort

boton committed:
LPS-121871 Clean old management-toolbar actions of DM AUI module

victorg1991 committed:
LPS-127459 Fix menu item styles

victorg1991 committed:
LPS-127459 Reset selected menu item when closing the sidebar

guipfreire committed:
LRQA-65270 Remove AppBuilder#PublishAppAsWidgetWithOnlyTableView from quarantine.

brianchandotcom committed:
LPS-124886 rename

brianchandotcom committed:
LPS-127933 Sort

guipfreire committed:
LRQA-65275 Remove AppBuilder#PublishAppsInOneOrMoreSites from quarantine.

brianchandotcom committed:
LPS-127986 Rename

cgoncas committed:
LPS-128238 Simplify

ealonso committed:
LPS-128282 baseline

guipfreire committed:
LRQA-65280 Remove AppBuilder#PublishAppsInProductMenuAndReuseItsViews.

michaelhashimoto committed:
LRCI-2004 Always set the slave size to 32 whenever testing analytics cloud

brianchandotcom committed:
COMMERCE-5731 SF

brianchandotcom committed:
COMMERCE-5731 bump

cgoncas committed:
LPS-128238 Maybe the LAYOUT_DISPLAY_PAGE_OBJECT_PROVIDER has not been initialized.

marcosapmf committed:
LPS-118239 take locale path param into account when getting default languageId

javiergamarra committed:
LPS-127775 allow new lines in descriptions

guipfreire committed:
LRQA-65286 Remove AppBuilder#PublishAppsInStandaloneAndReuseItsViews.

Commit:
LPS-126702 - Update @clayui dependencies to 3.25.0

Commit:
LPS-126702 - Update snapshots

Commit:
LRQA-65295 - Fix Reports tests

natocesarrego committed:
LRQA-65041 Do not ignore test in order to track its health on Testray

vicnate5 committed:
LRQA-65293 Use regular test batches instead of smoke for environment testing suite so that a single non-functional console warning doesn't fail most tests in this large suite that is focused on functional behavior

hhuijser committed:
LPS-125803 Log exception instead of swallowing

rodrigopaulino committed:
LPS-127602 Update editor configurations with the new generated instanceId

pyoo47 committed:
LRCI-2008 Implement getCurrentTimeMillis()

ealonso committed:
LPS-128289 Adds resources

ealonso committed:
LPS-128289 Adds new test to check if the link to layout value is converted to the correct format

ealonso committed:
LPS-128289 Moves code from JournalConverterImpl to JournalContentCompatibilityConverterImpl, to avoid converting it every time we save the web content.

ealonso committed:
LPS-128289 Don't format test-journal-content-link-to-page-field-compatibility-expected-results.xml since if we format this file we can not use Assert.assertEquals(expectedDocument.formattedString(), document.formattedString()); to compare it

ealonso committed:
LPS-128289 Don't format this file

ealonso committed:
LPS-128289 Removes unnecessary code since it is already added in TemplateNode._getLatestArticleData()

ruben-pulido committed:
LPS-128289 Remove testGetLinkToLayoutValue from JournalConverterUtilTest and replace it with testGetLinkToLayoutValue in JournalContentCompatibilityConverterTest

liuxingye committed:
LRQA-65265 Update path for open graph custom image input

pyoo47 committed:
LRCI-2008 Use JenkinsResultsParserUtil.getCurrentTimeMillis() in GitHubDevSyncUtil

joshchong committed:
LRQA-65298 Update logic for printing Elasticsearch indices

joshchong committed:
LRQA-65298 Print second cluster indices for CCR tests only

ruben-pulido committed:
LPS-128016 Return 0 for fileEntryId if classNameId is 0

liuxingye committed:
LPS-128178 Add test for view localized DM field

Commit:
LRQA-64845 Add macro to get Css Value

Commit:
LRQA-64845 Add macro to assert Css Value is greater than

Commit:
LRQA-64845 Add case to view Product Menu styles

Hanlf committed:
LRQA-65305 Update locator about Message Boards publish button

emanuelfisilva committed:
LRQA-65306 Update Data Engine functional relevant suite

liuxingye committed:
LPS-128247 Add path for the second select button

liuxingye committed:
LPS-128247 Add test for view repeatable image field when edit

liuxingye committed:
LPS-125783 Add paths for confirm delete message

liuxingye committed:
LPS-125783 Add macros for confirm delete message

liuxingye committed:
LPS-125783 Add tests for confirm message when delete questions

emanuelfisilva committed:
LRQA-65306 Add acceptance property on relevant tests

Hanlf committed:
LRQA-65303 Update locators about read only fields

Hanlf committed:
LRQA-65309 Update locator about Link#SUBSCRIBE

austinchiang committed:
LPS-128177 Add test for the document type localization

zhangyan0701 committed:
LRQA-65299 Add smoke test that compile JSPs at runtime

Hanlf committed:
LRQA-65313 Update macro to make it work for remote staging

Hanlf committed:
LRQA-65313 Click content field first to make element visible

jkappler committed:
LPS-127873 Upgrade Netty/Codec/HTTP from 4.1.50 to 4.1.59 because of known vulnerability

achaparro committed:
LPS-128098 Generate remote services only when needed

achaparro committed:
LPS-128098 Build services

izaera committed:
LPS-127990 Changed more references after rebase

brianchandotcom committed:
LPS-125404 rename

brianchandotcom committed:
LPS-128214 SF

carolmariaabb committed:
LPS-128291 Place the setting Maximum Repetitions for Upload Field before the guest upload field settings

brianchandotcom committed:
LPS-128235 SF

carolmariaabb committed:
LPS-128291 Rename the setting Maximum Submissions for Guest Upload Fields to Guest Upload Maximum Submissions

carolmariaabb committed:
LPS-128291 buildLang

guipfreire committed:
LRQA-65320 Remove SharedAppBuilder#DeleteFormViewFieldFromObject from quarantine.

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LPS-128289 SF

guipfreire committed:
LRQA-65321 Remove SharedAppBuilder#CreateRemoveUpdateDeleteEntriesOfProductMenuApp from quarantine.

brianchandotcom committed:
LRCI-2000 prep next

brianchandotcom committed:
LRCI-2008 prep next

jkappler committed:
LPS-128336 Only show experiences for pages of type content, otherwise retrieve and empty list

guipfreire committed:
LRQA-65322 Remove SharedAppBuilder#CreateTableViewsWithFixedEntriesFilter from quarantine.

Commit:
COMMERCE-5741 commerce screen-navigation usages updated to match new apis

brianchandotcom committed:
LPS-127775 prep next

brianchandotcom committed:
LPS-127775 prep next

Commit:
COMMERCE-5695 dataset events definitions updated to be consistent

Commit:
LPS-128283 Create WC smoke test

matuzalemsteles committed:
LPS-127671 Revert "Set minimum column size two 2 and adjust resizing"

This reverts commit 247cbe14bcde321b3a8f287585e76502139478cb.

We are reversing this implementation because it implements a behavior different from what would be expected, we must consider that the minimum column size must be only 2 but allow the resize to be done one by one. The way it was implemented, the user is unable to create a column with size 3, only two by two, that is, just even numbers

An important point is that we should test if the Form loads a form with non-even fields, the resize will correct and mess up the layout since it will only have even-sized columns.

brianchandotcom committed:
LRCI-2004 prep next

brianchandotcom committed:
Revert "LRCI-2004 Always set the slave size to 32 whenever testing analytics cloud"

This reverts commit 216c4109b536d896e150d25101ef06baa1ad0632.

brianchandotcom committed:
LRCI-2004 prep next (revert)

Commit:
LPS-128283 SF

Commit:
LRQA-59667 - Quarantine test

Commit:
LRQA-65324 Remove ignore

michaelhashimoto committed:
LRCI-2004 Explicitly set the Analytics Cloud tests to 32gb slaves

tinatian committed:
LPS-128360 Remove unnecessary log4j dependency for sample-sql-builder

hhuijser committed:
LPS-105380 Add logic to find redundant line breaks in constructor

hhuijser committed:
LPS-105380 Auto SF

vicnate5 committed:
LRQA-65299 Naming

vicnate5 committed:
LRQA-65299 Correct directory

Commit:
LRQA-65327 - Fix KaleoformsUsecase#ViewProcessWithComments test

emanuelfisilva committed:
LPS-126340 SF

vicnate5 committed:
LRQA-65299 Delete before starting app server

pei-jung committed:
LPS-125991 Semver

hhuijser committed:
LPS-105380 Combine literal string and StringPool.* in plus statements

hhuijser committed:
LPS-105380 Add logic to find cases where we can combine StringPool.* and literal string

xbrianlee committed:
LRQA-65329 Exclude document library when deleting data folder contents when HSQL database isn't cleared

mtambara committed:
LPS-128031 Enable test on master

BryanEngler committed:
LPS-128368 portal-search-web: Fix bad SF in 957f17b75adbeafc0efabdc13e666b4d58918e5a

https://issues.liferay.com/browse/LPS-128368

Commit:
LRQA-64845 Modify macro to use xpath

Commit:
LRQA-64845 Revert "LRQA-64845 Add macro to get Css Value"

This reverts commit 1f821999f26bf3681e764b6853511c04720327ec.

emanuelfisilva committed:
LPS-126851 Remove discarded test stub

Commit:
LRQA-64845 Move macro to Css file

Commit:
LRQA-64845 Use new macro

Commit:
LRQA-64845 Use new parameters

Commit:
LRQA-64770 Add Oracle to database type

Hanlf committed:
LRQA-65316 Fix issue about asserting user is removed

Commit:
LPS-127481 dispatch-service - wait for modules to initialize (not optimal but safe enough)

brianchandotcom committed:
LPS-127990 SF

brianchandotcom committed:
LPS-124594 Usual order

brianchandotcom committed:
LPS-128291 Rephrase

brianchandotcom committed:
LPS-128291 regen

brianchandotcom committed:
LPS-128254 Wordsmith

liferay-translation committed:
LPS-77699 Update Translations

brianchandotcom committed:
LPS-127777 Sort

brianchandotcom committed:
LPS-127777 prep next

brianchandotcom committed:
LPS-128098 prep next

brianchandotcom committed:
LPS-128098 LPS-127777 prep next
